hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/om/TestTransactionInfoRepair.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.hadoop.ozone.repair.ldb;
+package org.apache.hadoop.ozone.om;
```

Comment:
This should be moved to slightly different package:

```suggestion
package org.apache.hadoop.ozone.repair.om;
```